docs/docs/rest-api/public/api/v2/apps.raml

```diff
@@ -0,0 +1,451 @@
+
+###
+# Get a list of applications
+#
+get:
+  description: Get the list of running applications. Several filters can be applied to filter the result list.
+  is: [ secured ]
```

Comment:
As far as I can see, this leads to the output having a `Security Schemes` section with one `Anonymous` entry – this might be a little irritating, as it looks like _the anonymous security scheme is applied_ …